docs/admin/troubleshooting.md

```diff
@@ -433,6 +433,142 @@ ERROR: dmg: pool create failed: DER_NOSPACE(-1007): No space on storage target
 	$ dmg pool destroy --pool=$DAOS_POOL --force
 	Pool-destroy command succeeded
 ```
+
+## Diagnostic and Recovery Tools
+
+!!! WARNING : Please be careful and use this tool under supervision of DAOS export.
```

Comment:
"export"? do you mean "the DAOS support team"?

Reply:
Updating.